src/Command/Redis/BZPOPMAX.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Predis\Command\Redis;
+
+/**
+ * @link https://redis.io/commands/bzpopmax/
+ *
+ * BZPOPMAX is the blocking variant of the sorted set ZPOPMAX primitive.
+ *
+ * It is the blocking version because it blocks the connection when there are
+ * no members to pop from any of the given sorted sets.
+ * A member with the highest score is popped from first sorted set that is non-empty,
+ * with the given keys being checked in the order that they are given.
+ */
+class BZPOPMAX extends BZPOPMIN
```

Comment:
This doesn't seem right. Either we can have a BZPOPBase which would be extended by both BZPOPMAX and BZPOPMIN or these two would be independent.

Reply:
Good point

Reply:
@sazzad16 Fixed